app/assets/images/payment_icons/discover.svg

```diff
@@ -1 +1 @@
/><path d="M37 13s-9.89 6.904-28 10h25a3 3 0 003-3v-7z" fill="#F48120"/><path d="M4 .5h30A3.5 3.5 0 0137.5 4v16a3.5 3.5 0 01-3.5 3.5H4A3.5 3.5 0 01.5 20V4A3.5 3.5 0 014 .5z" stroke="#000" stroke-opacity=".07"/><path d="M3.57 7.16H2v5.5h1.57c.83 0 1.43-.2 1.96-.63.63-.52 1-1.3 1-2.11-.01-1.63-1.22-2.76-2.96-2.76zm1.26 4.14c-.34.3-.77.44-1.47.44h-.29V8.1h.29c.69 0 1.11.12 1.47.44.37.33.59.84.59 1.37 0 .53-.22 1.06-.59 1.39zm2.19-4.14h1.07v5.5H7.02v-5.5zm3.69 2.11c-.64-.24-.83-.4-.83-.69 0-.35.34-.61.8-.61.32 0 .59.13.86.45l.56-.73c-.46-.4-1.01-.61-1.62-.61-.97 0-1.72.68-1.72 1.58 0 .76.35 1.15 1.35 1.51.42.15.63.25.74.31.21.14.32.34.32.57 0 .45-.35.78-.83.78-.51 0-.92-.26-1.17-.73l-.69.67c.49.73 1.09 1.05 1.9 1.05 1.11 0 1.9-.74 1.9-1.81.02-.89-.35-1.29-1.57-1.74zm1.92.65c0 1.62 1.27 2.87 2.9 2.87.46 0 .86-.09 1.34-.32v-1.26c-.43.43-.81.6-1.29.6-1.08 0-1.85-.78-1.85-1.9 0-1.06.79-1.89 1.8-1.89.51 0 .9.18 1.34.62V7.38c-.47-.24-.86-.34-1.32-.34-1.61 0-2.92 1.28-2.92 2.88zm12.76.94l-1.47-3.7h-1.17l2.33 5.64h.58l2.37-5.64h-1.16l-1.48 3.7zm3.13 1.8h3.04v-.93h-1.97v-1.48h1.9v-.93h-1.9V8.1h1.97v-.94h-3.04v5.5zm7.29-3.87c0-1.03-.71-1.62-1.95-1.62h-1.59v5.5h1.07v-2.21h.14l1.48 2.21h1.32l-1.73-2.32c.81-.17 1.26-.72 1.26-1.56zm-2.16.91h-.31V8.03h.33c.67 0 1.03.28 1.03.82 0 .55-.36.85-1.05.85z" fill="#231F20"/><path d="M20.16 12.86a2.93 2.93 0 100-5.86 2.93 2.93 0 000 5.86z" fill="url(#pi-paint0_linear)"/><path opacity=".65" d="M20.16 12.86a2.93 2.93 0 100-5.86 2.93 2.93 0 000 5.86z" fill="url(#pi-paint1_linear)"/><path d="M36.5709 7.50593C36.5709 7.40593 36.5009 7.35593 36.3909 7.35593H36.2309V7.83593H36.3509V7.64593L36.4909 7.83593H36.6309L36.4709 7.63593C36.5309 7.62593 36.5709 7.57593 36.5709 7.50593V7.50593ZM36.3709 7.57593H36.3509V7.44593H36.3709C36.4309 7.44593 36.4609 7.46593 36.4609 7.50593C36.4609 7.55593 36.4309 7.57593 36.3709 7.57593Z" fill="#231F20"/><path d="M36.4109 7.17592C36.1809 7.17592 35.9909 7.36592 35.9909 7.59592C35.9909 7.82592 36.1809 8.01592 36.4109 8.01592C36.6409 8.01592 36.8309 7.82592 36.8309 7.59592C36.8309 7.36592 36.6409 7.17592 36.4109 7.17592ZM36.4109 7.94592C36.2309 7.94592 36.0709 7.79592 36.0709 7.59592C36.0709 7.40592 36.2209 7.24592 36.4109 7.24592C36.5909 7.24592 36.7409 7.40592 36.7409 7.59592C36.7409 7.78592 36.5909 7.94592 36.4109 7.94592Z" fill="#231F20"/><defs><linearGradient id="pi-paint0_linear" x1="21.657" y1="12.275" x2="19.632" y2="9.104" gradientUnits="userSpaceOnUse"><stop stop-color="#F89F20"/><stop offset=".25" stop-color="#F79A20"/><stop offset=".533" stop-color="#F68D20"/><stop offset=".62" stop-color="#F58720"/><stop offset=".723" stop-color="#F48120"/><stop offset="1" stop-color="#F37521"/></linearGradient><linearGradient id="pi-paint1_linear" x1="21.338" y1="12.232" x2="18.378" y2="6.446" gradientUnits="userSpaceOnUse"><stop stop-color="#F58720"/><stop offset=".359" stop-color="#E16F27"/><stop offset=".703" stop-color="#D4602C"/><stop offset=".982" stop-color="#D05B2E"/></linearGradient></defs></svg>
```

Comment:
There seems to be a border radius difference. I think it has to do with the border settings in the illustration tool where you can set between "inside" or "outside" options.

[video demonstration](https://screenshot.click/08-08-blrbh-ne5mo.mp4)

Reply:
Thanks @tauthomas01. Here's the latest iteration with an update to the border radius. I took the SVG elements from the Visa card to update this in Figma and re-added the attributes from the guidelines. Anything else to consider that I may be missing?

![](https://screenshot.click/08-46-n5cds-ubvg5.jpg)